tests/js/advancedCounter.scrypttest.js

```diff
@@ -23,7 +22,7 @@ const outputAmount = 222222
 const changeAmount = 111111
 
 describe('Test sCrypt contract Counter In Javascript', () => {
-  let counter, preimage, result
+  let counter, preimage, result, tx = newTx();
```

Comment:
Move to Line 11, same for every scrypttest.

Reply:
9a531461eca019f0a14ad12158cfe4e098d0e462